Doc/library/http.rst

```diff
@@ -171,16 +171,25 @@ Property             Indicates that           Details
    Usage::
 
       >>> from http import HTTPMethod
-      >>> HTTMethod.GET
-      HTTMethod.GET
-      >>> HTTMethod.GET == 'GET'
+      >>>
+      >>> HTTPMethod.GET
+      <HTTPMethod.GET>
+      >>> HTTPMethod.GET == 'GET'
       True
-      >>> HTTMethod.GET.value
+      >>> HTTPMethod.GET.value
       'GET'
-      >>> HTTMethod.GET.description
+      >>> HTTPMethod.GET.description
       'Transfer a current representation of the target resource.'
```

Comment:
```suggestion
      'Retrieve the target.'
```
https://github.com/python/cpython/blob/57be5459593bbd09583317ebdafc4d58ae51dbf4/Lib/http/__init__.py#L194

It's also this way on the 3.11 branch